src/main/java/org/assertj/core/error/AssertionErrorMessagesAggregrator.java

```diff
@@ -14,6 +14,10 @@
 
 import java.util.List;
 
+/**
+ * @deprecated use {@link AssertionErrorMessagesAggregator} instead
+ */
+@Deprecated
 public class AssertionErrorMessagesAggregrator {
 
   public static String aggregrateErrorMessages(List<String> errors) {
```

Comment:
Perhaps method from replacement class could be called here to stay DRY, and have no other code in this class?

Reply:
Done in 45eb16da4a2290a0d3f9c77f7806372507705af6, thanks @pzygielo!